client/webpack.config.js

```diff
@@ -0,0 +1,77 @@
+/* jshint esversion: 6 */
+const webpack = require('webpack');
+const path = require('path');
+
+const devBuild = process.env.NODE_ENV !== 'production';
+const nodeEnv = devBuild ? 'development' : 'production';
+
+const config = {
+  entry: [
+    'es5-shim/es5-shim',
```

Comment:
The minified es5 shim + sham is around 31kb. Is this insignificant enough for us to just leave it there?